cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4932,6 +4932,22 @@
     </description>
   </property>
 
+  <property>
+    <name>task.worker.http.client.read.timeout.ms</name>
```

Comment:
Shouldn't we use separate read timeout for configurator and basic service context? We can still use ${task.worker.http.client.read.timeout.ms} as value to not repeat the values, but it provides the benefit that we can set different timeouts.

Reply:
Separated the configs and kept the default property.